src/Poseidon/CLI/Forge.hs

```diff
@@ -90,7 +91,17 @@ runForge (ForgeOptions baseDirs entitiesDirect entitiesFile intersect outPath ou
     let [outInd, outSnp, outGeno] = case outFormat of 
             GenotypeFormatEigenstrat -> [outName <.> ".ind", outName <.> ".snp", outName <.> ".geno"]
             GenotypeFormatPlink -> [outName <.> ".fam", outName <.> ".bim", outName <.> ".bed"]
-    let newSNPSet = snpSetMergeList (map (snpSet . posPacGenotypeData) relevantPackages) intersect
+    -- output warning if any snpSet is set to Other
+    let snpSetList = map (snpSet . posPacGenotypeData) relevantPackages
+    let packageNamesWithSnpSetOther =
+            map snd . filter ((==SNPSetOther) . fst) . zip snpSetList . map posPacTitle $ relevantPackages
+    when (not . null $ packageNamesWithSnpSetOther) $ do
+        hPutStrLn stderr ("A friendly warning: One or more of your source packages (" ++ show packageNamesWithSnpSetOther ++
+            ") has snpSet: Other, which will result in your forged package to also have \
+            \snpSet: Other. If this is intended, please ignore this message. If not, \
+            \please enter the correct snpSet in those packages to make sure our forging can deduce \
+            \the right target snpSet")
+    let newSNPSet = snpSetMergeList snpSetList intersect
```

Comment:
I guess you could just use the output of `snpSetMergeList` for this check, right? It already has all you need to decide if the warning should be shown. `packageNamesWithSnpSetOther` is not necessary.

Reply:
Well, I would definitely want to keep `snpSetMergeList` pure, so outputting error is not possible. Do you mean to use the result of that function to see whether it's `SNPSetOther`? Or do you mean I should add to that function to return also those input packages with `Other`? I would like to avoid making `snpSetMergeList` more complex, I like the simplicity.

Reply:
Yes - indeed. I meant instead of checking if there is any `SNPSetOther` in the list of packages, you can as well just check if `snpSetMergeList` returns `SNPSetOther`. It's simpler, faster and should yield an identical result.

Reply:
yes, true. But for the error message to be useful, we do need the names of the culprits, so I'm afraid we still need this zipped `packageNamesWithSnpSetOther`, or do you have a better idea?

Reply:
Ok - sorry. I missed the `show packageNamesWithSnpSetOther`